### PR TITLE
[T3A3][W10-A4] Wang Zexin

### PR DESCRIPTION
**Pull request by wangzexin**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "a/123, Clementi Ave 3, #12-34, 231534";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses should be entered in the following format a/BLOCK, STREET, UNIT, POSTAL_CODE";
+    public static final String ADDRESS_VALIDATION_REGEX = ".+, .+, .+, .+";
 
-    public final String value;
```
Comment (hwkchia) [inline src/seedu/addressbook/data/person/Address.java:15]: Great! You removed this.

**Review by hwkchia — COMMENTED** [review]
Minimal review done here, as this PR was reviewed during the tutorial.  Just remember to adhere to code quality issues like commenting for the newly created classes.  Also did you change your user guide?

**Comment by hwkchia** [comment]
Ack by closing PR.

File: src/seedu/addressbook/data/person/Address.java (hunk 1 above)
Comment (wangzexin) [reply src/seedu/addressbook/data/person/Address.java:15]: Oh no... I think I forgot to change the user guide.
Will do so after finishing T4A7!

**Pull request closed without merging**